s3/compress.go

```diff
@@ -101,16 +99,17 @@ func (c *Compressor) zipExport(ctx context.Context, prefix, filename, s3key stri
 
 		fileMeta = append(fileMeta, *tempFileMeta)
 
-		header, err := tar.FileInfoHeader(fi, basename)
+		header, err := zip.FileInfoHeader(fi)
 		if err != nil {
 			return fmt.Errorf("failed to create file header: %w", err)
 		}
 		header.Name = basename
 
-		if err = tarWriter.WriteHeader(header); err != nil {
+		var zippedFile io.Writer
+		if zippedFile, err = zipWriter.CreateHeader(header); err != nil {
```

Comment:
Do we even need the above? 
```suggestion
		if zippedFile, err := zipWriter.CreateHeader(header); err != nil {
```
Would this not work?

Reply:
No, that approach will not work as it is currently coded.  With the approach you are suggesting, the `zippedFile` variable will be scoped to that if statement, but the `zippedFile` variable is needed outside of that if statement.

Another approach, would be to split the if statement like this:
`
                zippedFile, err := zipWriter.CreateHeader(header); 
		if err != nil {
`